### ISSUE TITLE
EncodedData Wikipedia reference is in Norwegian

### ISSUE THREAD

**Issue by jsimonclark** [emmo-repo/EMMO#232]
The wikipedia reference for EncodedData term in perspectives/data.ttl should be "https://en.wikipedia.org/wiki/Data" so that it resolves in English rather than Norwegian.

**Comment by jesper-friis** [comment]
Yes, that would make sense. However, the description of "data" on Wikipedia differ a lot between the English and Norwegian pages. EMMO relies on the description of data by Floridi, which is only described on the Norwegian page.
